scheduler/pkg/util/model.go

```diff
@@ -36,3 +36,15 @@ func GetOrignalModelNameAndVersion(versionedModel string) (string, uint32, error
 func GetPinnedModelVersion() uint32 {
 	return 1
 }
+
+// if min and max replicas are not set, we do not allow autoscaling
```

Comment:
just to note that this is just for our internal model autoscaling using lag. in the future we also do not want to make the decision whether autoscaling is enabled by check `minReplicas` and `maxReplicas` (as a future PR).

Reply:
yep, that makes sense.